daemon/main.go

```diff
@@ -83,6 +86,61 @@ func main() {
 	}
 }
 
+func checkMinRequirements() {
+	log.Infof("Checking minimal requirements...")
+	clangVersion, err := exec.Command("clang", "--version").CombinedOutput()
+	if err != nil {
+		log.Fatalf("clang 3.8.x version: NOT OK: %s", err)
+	}
+	if !strings.Contains(string(clangVersion), "3.8") {
+		log.Fatalf("clang 3.8.x version: NOT OK, please install clang version 3.8.x in your system")
+	}
+	log.Infof("clang 3.8.x version: OK!")
+	lccVersion, err := exec.Command("llc", "--version").CombinedOutput()
+	if err == nil {
+		if strings.Contains(strings.ToLower(string(lccVersion)), "debug") {
+			log.Warningf("llc version was compiled in debug mode, expect higher latency!")
+		}
+	}
+	// /usr/include/gnu/stubs-32.h is installed by 'glibc-devel.i686' in fedora
+	// /usr/include/sys/cdefs.h is installed by 'libc6-dev-i386' in ubuntu
+	// both files exist on both systems but cdefs.h already exists in fedora
+	// without 'glibc-devel.i686' so we check for 'stubs-32.h first.
+	if _, err := os.Stat("/usr/include/gnu/stubs-32.h"); os.IsNotExist(err) {
```

Comment:
Using `find` is too expensive. Let's go with what you have and if we get complaints we can add additional paths or convert it into a warning.